nuget/helpers/lib/NuGetUpdater/.editorconfig

```diff
@@ -24,6 +24,8 @@ insert_final_newline = true
 #### .NET Coding Conventions ####
 [*.{cs,vb}]
 
+max_line_length = 180
```

Comment:
We do not want to enforce this as not all tooling supports it. Also, it would require working out appropriate configuration for how to correctly break long lines.

Reply:
When I remove it it will fall to 80 characters (default). Which starts to change the code

Reply:
If this works as other .editorconfig properties, then I believe you could use `max_line_length = unset`.

Reply:
I will try that

Reply:
Also, it seems some editor allow you to set the value to `0` to mean no limit.

Reply:
The `unset` option results in an 80 (or default) character limit. The `0` seems it disable the max line length rule